design/account.go

```diff
@@ -298,6 +322,29 @@ var userDataAttributes = a.Type("UserDataAttributes", func() {
 	})
 })
 
+// showUserResources a list of resources in which the user has a role
+var showUserResources = JSONList(
+	"UserResources", "Holds the paginated response to a user spaces request",
+	userResourceData,
+	pagingLinks,
+	userListMeta)
+
+// userResourceData represents a resource for which a user has a role
+var userResourceData = a.Type("UserResourceData", func() {
+	a.Attribute("id", d.String, "id of the resource that in which the user has a role")
+	a.Attribute("type", d.String, "type of the resource")
+	a.Attribute("attributes", userResourceDataAttributes, "Info about the role and scopes that the user has in the resource")
+	a.Attribute("links", genericLinks)
+	a.Required("id", "type", "attributes")
+})
+
+// userResourceDataAttributes contains info about the role and scopes that the user has in the resource
+var userResourceDataAttributes = a.Type("UserResourceDataAttributes", func() {
+	a.Attribute("role", d.String, "The role of the user in the corresponding resource")
+	a.Attribute("scopes", a.ArrayOf(d.String), "The scopes associated with the role of the user in the corresponding resource")
+	a.Required("role", "scopes")
+})
+
 // updateidentityDataAttributes represents an identified user object attributes used for updating a user.
 var updateUserDataAttributes = a.Type("UpdateIdentityDataAttributes", func() {
```

Comment:
variable `updateUserDataAttributes` has different name in documentation. Can you please change it?

Reply:
sure, let me fix that